aten/src/ATen/native/cuda/Loops.cuh

```diff
@@ -3,7 +3,13 @@
 
 #include <ATen/detail/FunctionTraits.h>
 
-namespace at { namespace native { namespace modern { namespace detail {
+namespace at { namespace native {
+
+constexpr int num_threads = C10_WARP_SIZE * 2;
```

Comment:
I assume those values are unchanged from what they used to be for ROCm? And at present we don't need to specialize them on a per-kernel basis? If so, then constexp'ing them rather than having them as template arguments makes sense.